src/formula1_race_analysis/models.py

```diff
@@ -0,0 +1,29 @@
+from dataclasses import dataclass
+from datetime import timedelta
+
+
+@dataclass(order=True)
+class Driver:
+    identifier: str
+    name: str
+    car_model: str
+
+
+@dataclass
+class RaceResults:
```

Comment:
I would say it is just Result. Results in plural means there is more than one result in it